editor/components/writing-flow/index.js

```diff
@@ -70,8 +70,14 @@ class WritingFlow extends Component {
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.bindContainer = this.bindContainer.bind( this );
 		this.clearVerticalRect = this.clearVerticalRect.bind( this );
-		this.focusLastTextField = this.focusLastTextField.bind( this );
```

Comment:
Guessing it was a faulty rebase, but this line should not have been removed. It broke the "click below canvas" behavior.

Reply:
Hm, I'm sorry, not sure how this happened.

Reply:
It was a good excuse to add an E2E test 😄 #5991